app/src/main/java/org/oppia/android/app/profile/ProfileInputView.kt

```diff
@@ -61,7 +61,8 @@ class ProfileInputView @JvmOverloads constructor(
     if (
       attributes.getBoolean(
         R.styleable.ProfileInputView_isPasswordInput,
-        /** defVal= */ false
+        /** defVal= */
+        false
```

Comment:
One last thing from my side, could you make these in one line?

Reply:
Sure, will do!

Reply:
Done!

Reply:
This doesn't appear done yet--please fix.